src/adapter-manager.js

```diff
@@ -333,53 +333,58 @@ class AdapterManager extends EventEmitter {
   }
 
   /**
-   * @method loadAdapter
+   * @method loadAdaptersByPackageName
    *
-   * Loads a single adapter.
+   * Loads adapters with the given package name.
    *
-   * @returns A promise which is resolved when the adapter is loaded.
+   * @param {String} packageName The package name of the adapters to load.
+   * @returns A promise which is resolved when the adapters are loaded.
    */
 
-  async loadAdapter(adapterId) {
+  async loadAdaptersByPackageName(packageName) {
     const adapterPath = path.join(__dirname,
                                   config.get('adapterManager.path'),
-                                  adapterId);
+                                  packageName);
     const testMode = process.env.NODE_ENV === 'test';
 
     // Skip if there's no package.json file.
     const packageJson = path.join(adapterPath, 'package.json');
     if (!fs.lstatSync(packageJson).isFile()) {
-      console.error('package.json not found for adapter:', adapterId);
-      return Promise.reject();
+      const err = `package.json not found for package: ${packageName}`;
```

Comment:
While updating some other code, I noticed that this isn't using single quotes but some weird unicode character. And if you remove the package.json file and try to execute this path, it errors out.